packages/babel-helpers/scripts/generate-helpers.js

```diff
@@ -0,0 +1,60 @@
+import fs from "fs";
```

Comment:
nit: we can just import `fs/promises`.

Reply:
No because we build on Node 12 for the Babel 8 tests